angular-route/templates/_.client.route.js

```diff
@@ -0,0 +1,5 @@
+$stateProvider.
+  state('<%= slugifiedName %>', {
+    url: '/<%= slugifiedRoutePath %>',
+    templateUrl: 'modules/<%= slugifiedModuleName %>/client/views/<%= slugifiedViewName %>.client.view.html'
+  }).
```

Comment:
Where is this implemented? How does it work? What is the goal and the end result?